### DISCUSSION TITLE
Cannot use JSON to authenticate to the admin API using cURL

### DISCUSSION THREAD

**Discussion by AliSajid** [pocketbase/pocketbase#1091] [Q&A] [answered]
I have been playing with loading some data into my pocketbase instance. I just upgraded to 0.8.0. Once upgrade, I cannot use JSON to call and authenticate my admin to the pocketbase instance.

I get the following error in the pocketbase logs:

```
{
  "errorDetails": "code=400, message=Unmarshal type error: expected=forms.AdminLogin, got=string, field=, offset=100, internal=json: cannot unmarshal string into Go value of type forms.AdminLogin",
  "errorMessage": "An error occurred while loading the submitted data."
}
```

I get this response in my terminal:

```
{
    "code": 400,
    "message": "An error occurred while loading the submitted data.",
    "data": {}
}
```

**Comment by ganigeorgiev** [comment, marked as answer]
_I assume that you've migrated your pb_data from v0.7.x -> v0.8.0 [following the migration guide](https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md#migrate-from-v07x)._

Could you provide a code sample how you are sending the request?

**Reply by ganigeorgiev** [reply]
@AliSajid I cannot reproduce the error:

```
curl -X POST \
  https://pocketbase.io/api/admins/auth-with-password \
  -H 'content-type: application/json' \
  -d '{
  "identity": "test@example.com",
  "password": "123456"
}'
```

Make sure that your curl data payload is a valid json.

**Reply by AliSajid** [reply]
I have completed the migration so that is clean.

Here's what I am running with some information redacted.

```
curl --request POST --url https://pbfly.fly.dev/api/admins/auth-with-password --data "{'identity': 'MYEMAIL', 'password': 'MYPASSWORD'}" -H 'Content-Type: application/json'
```

**Reply by ganigeorgiev** [reply]
@AliSajid The json data is not properly formatted - you need to use double quotes for the json key and string values:

```
curl --request POST --url https://pbfly.fly.dev/api/admins/auth-with-password --data '{"identity": "MYEMAIL", "password": "MYPASSWORD"}' -H 'Content-Type: application/json'
```

**Reply by AliSajid** [reply]
🤦🏽  That worked. Thank you.
